libraries/AP_Baro/AP_Baro_Backend.h

```diff
@@ -50,10 +50,11 @@ class AP_Baro_Backend
         DEVTYPE_BARO_ICP101XX = 0x0F,
         DEVTYPE_BARO_ICP201XX = 0x10,
         DEVTYPE_BARO_MS5607   = 0x11,
-        DEVTYPE_BARO_MS5837   = 0x12,
-        DEVTYPE_BARO_MS5637   = 0x13,
-        DEVTYPE_BARO_BMP390   = 0x14,
-        DEVTYPE_BARO_BMP581   = 0x15,
+        DEVTYPE_BARO_MS5837_30BA = 0x12, // New type for MS5837-30BA
+        DEVTYPE_BARO_MS5837_02BA = 0x13, // New type for MS5837-02BA
+        DEVTYPE_BARO_MS5637   = 0x14,
+        DEVTYPE_BARO_BMP390   = 0x15,
+        DEVTYPE_BARO_BMP581   = 0x16,
```

Comment:
```suggestion
        DEVTYPE_BARO_MS5837_30BA = 0x12,
        DEVTYPE_BARO_MS5637   = 0x13,
        DEVTYPE_BARO_BMP390   = 0x14,
        DEVTYPE_BARO_BMP581   = 0x15,
        DEVTYPE_BARO_MS5837_02BA = 0x16,
```
It's better not to change existing device IDs, to avoid making current software versions report incorrect device types, and avoid new software needing to keep track of which firmware version they're talking to to know which set of values are in use.